pkg/bootstrap/api/helpers.go

```diff
@@ -32,3 +34,19 @@ func ValidateBootstrapGroupName(name string) error {
 	}
 	return fmt.Errorf("bootstrap group %q is invalid (must match %s)", name, BootstrapGroupPattern)
 }
+
+// ValidateUsages validates that the passed in string are valid usage strings for bootstrap tokens.
+func ValidateUsages(usages []string) error {
+	usageAuthentication := strings.TrimPrefix(BootstrapTokenUsageAuthentication, BootstrapTokenUsagePrefix)
+	usageSigning := strings.TrimPrefix(BootstrapTokenUsageSigningKey, BootstrapTokenUsagePrefix)
+	invalidUsages := sets.NewString()
+	for _, usage := range usages {
+		if usage != usageAuthentication && usage != usageSigning {
+			invalidUsages.Insert(usage)
+		}
+	}
+	if len(invalidUsages) > 0 {
+		return fmt.Errorf("invalide bootstrap token usage string: %s, valid usage option: %s, %s", strings.Join(invalidUsages.List(), ","), usageAuthentication, usageSigning)
```

Comment:
Ideally this would read as "valid usage options" as we are listing multiple. But Since we are trying to get this in I'll let it go.  Feel free to submit another quick PR to update that after this goes in if you like.